config/config_test.go

```diff
@@ -171,6 +171,17 @@ var _ = Describe("Test full application config", func() {
 		})
 	})
 
+	Context("configure zmq", func() {
+		It("returns an error if zmq isn't included", func() {
```

Comment:
can you also add a test for a working config?

Reply:
I copied what the google pubsub test did, LMK if this is what you were thinking